### ISSUE TITLE
Live test message broadcaster

### ISSUE THREAD

**Issue by cgewecke** [animist-io/whale-island#52]
Figure out which logic to use for the events.js `changeBroadcast` method. Either stop and restart or simply set service. Last step of #17 (Mechanism for contract to broadcast unique signal).

**Comment by cgewecke** [comment]
Closing in favor of #55 (Publications refactor) - that work needs to be done first.
